Mathlib/NumberTheory/Dioph.lean

```diff
@@ -518,7 +518,7 @@ theorem diophFn_vec_comp1 {S : Set (Vector3 ℕ (succ n))} (d : Dioph S) {f : Ve
 #align dioph.dioph_fn_vec_comp1 Dioph.diophFn_vec_comp1
 
 theorem vec_ex1_dioph (n) {S : Set (Vector3 ℕ (succ n))} (d : Dioph S) :
-    Dioph {v : Fin2 n → ℕ | ∃ x, (x::v) ∈ S} :=
```

Comment:
Similarly here, you should not need to change this

Reply:
It was very strange. Originally The file Mathlib/NumberTheory/Dioph.lean has some errors. Lean4 gave me some suggestions on how to change the codes. Hence I followed the suggestions. But then there were some merge conflicts.